--- main.cpp	2021-01-25 14:53:09.017136809 +0100
+++ "main copy.cpp"	2021-01-26 17:59:05.694205478 +0100
@@ -232,7 +232,7 @@
         slog::info << "Start inference " << slog::endl;
 
         bool isLastFrame = false;
-        bool isAsyncMode = false;  // execution is always started using SYNC mode
+        bool isAsyncMode = true;
         bool isModeChanged = false;  // set to TRUE when execution mode is changed (SYNC<->ASYNC)
 
         typedef std::chrono::duration<double, std::ratio<1, 1000>> ms;
@@ -345,7 +345,7 @@
                     }
                 }
             }
-            cv::imshow("Detection results", curr_frame);
+            //cv::imshow("Detection results", curr_frame);
 
             t1 = std::chrono::high_resolution_clock::now();
             ocv_render_time = std::chrono::duration_cast<ms>(t1 - t0).count();
